universal-application-tool-0.0.1/app/views/style/Styles.java

```diff
@@ -2477,8 +2513,8 @@ public final class Styles {
   public static final String TEXT_OPACITY_95 = "text-opacity-95";
   public static final String TEXT_OPACITY_100 = "text-opacity-100";
   public static final String TRUNCATE = "truncate";
-  public static final String OVERFLOW_ELLIPSIS = "overflow-ellipsis";
-  public static final String OVERFLOW_CLIP = "overflow-clip";
+  public static final String TEXT_ELLIPSIS = "text-ellipsis";
```

Comment:
[suggestion only - not a blocker] In the future, I'd suggest makeing these css changes in a seperate PR than the regex stuff, just to make it easier to review

Reply:
This is from the [upgrade guide](https://tailwindcss.com/docs/upgrade-guide#overflow-clip-ellipsis) for upgrading TailwindCSS v2 -> v3. Does that make a difference?

Reply:
Sorry! I see what you mean now. Yeah those renamed styles aren't used anywhere so putting this in a different PR would be better

Yeah theres waay too much stuff in this PR right now!